recipes/aws-noaa-solar-irradiance-daily/meta.yaml

```diff
@@ -0,0 +1,24 @@
+title: 'AWS NOAA SOLAR IRRADIANCE DAILY'
+description: 'Analysis-ready datasets derived from NOAA SOLAR IRRADIANCE DAILY'
+pangeo_forge_version: '0.9.2'
+pangeo_notebook_version: '2021.07.17'
+recipes:
+  - id: aws-noaa-solar-irradiance-daily
+    object: 'recipe:recipe'
+provenance:
+  providers:
+    - name: 'AWS NOAA Solar irradiance daily'
+      description: 'Registry of Open Data on AWS National Oceanographic & Atmospheric Administration National Centers for Environmental Information'
+      roles:
+        - producer
+        - licensor
+      url: s3://noaa-cdr-total-solar-irradiance-pds/
+  license: 'Open Data'
+maintainers:
+  - name: 'Ruben L. Mendoza'
+    orcid: '0000-0001-5880-7243'
+    github: Rub21
+bakery:
+  id: 'pangeo-noaa-cdr-total-solar-irradiance-pds'
```

Comment:
I think this will change eventually when @sharkinsspatial works through some things